flow/FileTraceLogWriter.cpp

```diff
@@ -111,6 +116,7 @@ void FileTraceLogWriter::open() {
 		}
 		else {
 			fprintf(stderr, "ERROR: could not create trace log file `%s' (%d: %s)\n", finalname.c_str(), errno, strerror(errno));
+			issues->addAndExpire("trace_log_could_not_create_file");
```

Comment:
I did a little digging, and apparently we already have a mechanism for gettin these errors in status. In some ways I prefer using the more standard issues to do it, except that they don't provide any details about the file that failed like the current mechanism. I propose that we only have one, so maybe we should keep the current one? Either that or try to get more custom details into the message. 

See https://github.com/apple/foundationdb/blob/7b6eff71011dbe7e66344cc1f9c88dc23c5638fe/fdbserver/Status.actor.cpp#L586 for where this is done. It's based off the trace event being logged below.

Reply:
If we keep the issues based approach, I think it should be the case that the issue persists indefinitely until the file is successfully opened (i.e. that you can't get a status that doesn't have the issue). I can't quite tell whether the expiration approach accomplishes that or not due to the retry behavior, but if it doesn't we may need to rethink it a bit.

Reply:
@etschannen What do you think? It seems like reusing the existing trace even based reporting way is easier

Reply:
We have decided to keep the current implementation and create another issue tracking the double logging from the old mechanism and remove it later.